### ISSUE TITLE
AttributeError: type object 'tqdm' has no attribute 'get_new'

### ISSUE THREAD

**Issue by neomatrix369** [tqdm/tqdm#1084]
- [ ] I have marked all applicable categories:
    + [X] exception-raising bug
    + [ ] visual output bug
    + [ ] documentation request (i.e. "X is missing from the documentation." If instead I want to ask "how to use X?" I understand [StackOverflow#tqdm] is more appropriate)
    + [ ] new feature request
- [X] I have visited the [source website], and in particular read the [known issues]
- [X] I have searched through the [issue tracker] for duplicates
- [X] I have mentioned version numbers, operating system and environment

--- 

While running this on a Kaggle Kernel I got this error, below are the details of the version of `tqdm` being used along with the system info.

### System + tqdm version info

```
4.53.0 3.7.6 | packaged by conda-forge | (default, Mar 23 2020, 23:03:20) 
[GCC 7.3.0] Linux
```

### Error message (+call stack)

```python
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<timed exec> in <module>

/kaggle/usr/lib/nlp_profiler_class/nlp_profiler_class.py in apply_text_profiling(self, dataframe, text_column, params)
     94         """
     95 
---> 96         return nlp_profiler.apply_text_profiling(dataframe, text_column, params)

/opt/conda/lib/python3.7/site-packages/nlp_profiler/core.py in apply_text_profiling(dataframe, text_column, params)
     60             actions_mappings.remove(item)
     61 
---> 62     apply_profiling_progress_bar = get_progress_bar(actions_mappings)
     63     for _, (param, action_description, action_function) in \
     64             enumerate(apply_profiling_progress_bar):

/opt/conda/lib/python3.7/site-packages/nlp_profiler/generate_features/parallelisation_methods/__init__.py in get_progress_bar(values)
     18 
     19 def get_progress_bar(values: list) -> tqdm:
---> 20     return tqdm(values, ncols=PROGRESS_BAR_WIDTH)
     21 
     22 

/opt/conda/lib/python3.7/site-packages/tqdm/asyncio.py in __new__(cls, *args, **kwargs)
     64 
     65     def __new__(cls, *args, **kwargs):
---> 66         return cls.get_new(super(tqdm_asyncio, cls), std_tqdm, *args, **kwargs)
     67 
     68 

AttributeError: type object 'tqdm' has no attribute 'get_new'
```

Here are the source files for [NLP Profiler](https://github.com/neomatrix369/nlp_profiler) referred above in the call stack:
- https://github.com/neomatrix369/nlp_profiler/blob/master/nlp_profiler/core.py
  - https://github.com/neomatrix369/nlp_profiler/blob/master/nlp_profiler/generate_features/__init__.py
     - https://github.com/neomatrix369/nlp_profiler/blob/master/nlp_profiler/generate_features/parallelisation_methods/__init__.py

Any reason this could be happening?

**Comment by neomatrix369** [comment]
I may have narrowed down this issue to `tqdm` versions after `4.46`, as the notebooks run fin up to this version. It's only one of the later versions causes issues, i.e. `4.52.0` or `4.53.0`

**Comment by casperdcl** [comment]
hmm interesting; does this only happen in a kaggle kernel?

**Comment by casperdcl** [comment]
should be fixed in `tqdm>=4.54.0`

**Comment by neomatrix369** [comment]
> hmm interesting; does this only happen in a kaggle kernel?

I haven't tried outside kaggle yet but I will do that and report, I'm pretty sure it's happening when 0.46.0 isn't found and it downloads the latest. I have pinned my library to 0.46.0 for now.

**Comment by neomatrix369** [comment]
> should be fixed in `tqdm>=4.54.0`

Thanks for the quick response.
